src/legacy/core_plugins/kibana/common/field_formats/types/boolean.ts

```diff
@@ -17,11 +17,19 @@
  * under the License.
  */
 
-import { asPrettyString } from '../../../../../../plugins/data/common/field_formats';
+import {
+  FieldFormat,
+  asPrettyString,
+  KBN_FIELD_TYPES,
+} from '../../../../../../plugins/data/common';
 
-export function createBoolFormat(FieldFormat) {
+export function createBoolFormat() {
   return class BoolFormat extends FieldFormat {
-    _convert(value) {
+    static id = 'boolean';
+    static title = 'Boolean';
+    static fieldType = [KBN_FIELD_TYPES.BOOLEAN, KBN_FIELD_TYPES.NUMBER, KBN_FIELD_TYPES.STRING];
+
+    _convert(value: any): string {
```

Comment:
could we start using `_convert` object already ?
`_convert: { text: (value: any): string { ... } }`

Reply:
Agree with you, all formatters will be refactored using this format.

Reply:
Yes, already can